app/components/work_package_relations_tab/index_component.html.erb

```diff
@@ -57,6 +57,7 @@
           # Combine visible and invisible relations into a single list
           all_relations = relation_group.visible_relations.map { |r| [r, :visible] } +
             relation_group.ghost_relations.map { |r| [r, :ghost] }
+          all_relations.sort_by! { |r| r[0].to.created_at }
```

Comment:
I wonder if ordering by related work package creation date is actually correct. I would expect a newly added work package to be listed at the bottom of the list (like before the relations tab redesign).

This could be achieved if we sort by `r[0].id` instead of `r[0].to.created_at` (because Relation does not have any `created_at`).

I think that's worth asking at the next UI/UX meeting.

On the other hand, ordering by relation creation date is something we can't achieve with child relations, unless we start introducing a `parent_updated_at` field.